src/test/java/com/qa/restassured/GetResponse.java

```diff
@@ -0,0 +1,21 @@
+package com.qa.restassured;
+
+import io.restassured.response.Response;
+
+import static io.restassured.RestAssured.given;
+
+public class GetResponse {
+    @org.testng.annotations.Test
```

Comment:
Curious why are we using whole package name instead just @Test ?

Reply:
Modify